content/misc/glossary/contents.lr

```diff
@@ -86,14 +89,19 @@ In Tor terms, a single document compiled and voted on by the [directory authorit
 
 ### cookie
 
-An [HTTP](#http) cookie (also called web cookie, Internet cookie, browser cookie or simply cookie) is a small piece of data sent from a website and stored on the user's computer by the user's [web browser](#web-browser) while the user is browsing. [Tor Browser](#tor-browser) does not store cookies.
+An [HTTP](#http) cookie (also called web cookie, Internet cookie, browser cookie or simply cookie) is a small piece of data sent from a website and stored on the user's computer by the user's [web browser](#web-browser) while the user is browsing. 
+By default, [Tor Browser](#tor-browser) does not store cookies. You can change this after visiting "preferences" (accessible from the hamburger menu at the top-right of the screen) and navigating to "Privacy & Security".
+
 ### cross-site scripting (XSS)
 
 Cross-Site Scripting (XSS) allows an attacker to add malicious functionality or behavior to a website when they shouldn't have the ability to do so.
 
 ### cryptographic signature
 
-A cryptographic signature proves the authenticity of a message or file. It is created by the holder of the private portion of a [public key cryptography](#public-key-cryptography) key pair and can be verified by the corresponding public key. If you download software from torproject.org, you will find it as signature files (.asc). These are PGP signatures, so you can verify that the file you have downloaded is exactly the one that we intended you to get. For more information about how you can verify signatures, please see [here](https://support.torproject.org/tbb/how-to-verify-signature/).
+A cryptographic signature proves the authenticity of a message or file. It is created by the holder of the private portion of a [public key cryptography](#public-key-cryptography) key pair and can be verified by the corresponding public key. 
+If you download software from torproject.org, you will find it as signature files (.asc). 
+These are PGP signatures, so you can verify that the file you have downloaded is exactly the one that we intended you to get. 
+For more information about how you can verify signatures, please see [here](https://support.torproject.org/tbb/how-to-verify-signature/).
```

Comment:
Instead of using [here], we could change the phrase to:
For more information, see [how you can verify signatures](https://support.torproject.org/tbb/how-to-verify-signature/).